tests/test-notebook/src/actions.spec.ts

```diff
@@ -916,6 +916,12 @@ describe('@jupyterlab/notebook', () => {
         expect(widget.isSelected(widget.widgets[0])).to.equal(true);
       });
 
+      it('should extend the selection to the topmost cell', () => {
+        widget.activeCellIndex = 1;
+        NotebookActions.extendSelectionAbove(widget, true);
+        expect(widget.isSelected(widget.widgets[0])).to.equal(true);
```

Comment:
Thanks! I think we're almost there!

This test goes to the second cell, then asks to select to the top, and checks that the one cell above it is selected. But that doesn't distinguish between the normal extendSelectionAbove behavior and extending to the very top. Can you go down to the bottom cell, and then check to see if every cell above it is selected? That would be testing that it actually is doing what it says, different than the behavior before.